### PR TITLE
Add new 3D Hybrid bubble example 

### PR DESCRIPTION
**Pull request by akshaysridhar**
2D Spectral (Horizontal) + 1D FiniteDifference (Vertical)
Build Function spaces
Updates b.c. to include UV Vectors for horizontal components
	new file:   bubble_3d.jl
Co-authored by: @szy21

### CODE REVIEW

File: examples/hybrid/bubble_3d.jl [hunk 1]
```diff
@@ -0,0 +1,284 @@
...
+    # Horizontal momentum
+    @. dYc.ρuₕ += -uvdivf2c(ρw ⊗ If(uₕ))
+    Ih = Ref(
+        Geometry.Axis2Tensor(
+            (Geometry.UVAxis(), Geometry.UVAxis()),
+            @SMatrix [1.0 0.0; 0.0 1.0]
+        ),
+    )
+    @. dYc.ρuₕ -= hdiv(Yc.ρuₕ ⊗ uₕ + p * Ih)
+    # ------------------->
```
Comment (szy21) [inline examples/hybrid/bubble_3d.jl:216]:
```suggestion
```

File: examples/hybrid/bubble_3d.jl [hunk 2]
```diff
@@ -0,0 +1,284 @@
...
+);
+
+ENV["GKSwstype"] = "nul"
+#import Plots
+#Plots.GRBackend()
+
+#dirname = "bubble_3d"
+#path = joinpath(@__DIR__, "output", dirname)
+#mkpath(path)
+#TODO: Slab plots + VTK output support for 3D domains
```
Comment (valeriabarra) [inline examples/hybrid/bubble_3d.jl:284]: Are these stale comments? Or meant to be decommented?

Comment (jakebolewski) [reply examples/hybrid/bubble_3d.jl:284]: we don't have visualization support for 3D extruded fields

**Review by jakebolewski — APPROVED** [review]

**Pull request merged**